serverless/develop/serverless-autoscaling-developer.adoc

```diff
@@ -1,17 +1,29 @@
-[id="serverless-autoscaling-scale-bounds"]
-= Scale bounds
+[id="serverless-autoscaling-developer"]
+= Autoscaling
 include::modules/common-attributes.adoc[]
 include::modules/serverless-document-attributes.adoc[]
-:context: serverless-autoscaling-scale-bounds
+:context: serverless-autoscaling-developer
 
 toc::[]
 
+Knative Serving provides automatic scaling, or _autoscaling_, for applications to match incoming demand. For example, if an application is receiving no traffic, and scale to zero is enabled, Knative Serving scales the application down to zero replicas. If scaling to zero is disabled, the application is scaled down to the minimum number of replicas specified for applications on the cluster. Replicas can also be scaled up to meet demand if traffic to the application increases.
+
+To enable autoscaling for Knative Serving, you must configure concurrency and scale bounds for your application.
```

Comment:
Afaik there are defaults for concurrency and scale bounds so you dont need to set them in order to enable autoscaling. [It would be more accurate to say](https://knative.dev/docs/serving/autoscaling/scale-to-zero/#configuring-scale-to-zero): "Assuming autoscaling is globally enabled then you can configure per service its required concurrency and scale bounds".

Reply:
I wondered about this exact thing! This was written a while back and I think maybe came from upstream (?) but this sounds MUCH better, thanks.